### PR TITLE
blkpr: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** util-linux 2.40

TODO:
- [x] Add an example for the "release" command.

### CODE REVIEW

File: pages/linux/blkpr.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# blkpr
+
+> Register, reserve, release, preempt, and clear persistent reservations on a block device that supports Persistent Reservations.
+> More information: <https://manned.org/blkpr>.
+
+- Register ([c]ommand) a new reservation with a given [k]ey on a given device:
+
+`blkpr {{-c|--command}} register --key {{reservation_key}} {{path/to/device}}`
+
+- Set the [t]ype an existing reservation to exclusive access:
```
Comment (tricantivu) [inline pages/linux/blkpr.md:10]:
```suggestion
- Set the [t]ype of an existing reservation to exclusive access:
```

File: pages/linux/blkpr.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`blkpr -c reserve -k {{reservation_key}} {{-t|--type}} exclusive-access {{path/to/device}}`
+
+- Preempt the existing reservation with a specific [K]ey and replaces it with a new reservation:
```
Comment (tricantivu) [inline pages/linux/blkpr.md:14]:
```suggestion
- Preempt the existing reservation with a specific [K]ey and replace it with a new reservation:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by tricantivu — APPROVED** [review]

**Comment by osbm** [comment]
This PR completed a step of #11827

**Pull request merged**